bigbluebutton-html5/imports/ui/components/loader/component.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
```

Comment:
The link @OZhurbenko explains why we need React then, JSX needs React on scope, as said in one of the comments